### ISSUE TITLE
Interoperate PolkaViz and PolkaAnalytics 

### ISSUE THREAD

**Issue by prastut** [thevantageproject/PolkaAnalytics#47]
I would want people to see relay chain bird's eye view on PolkaAnalytics and vice versa, specific views on PolkaViz are lagging behind now.

**Comment by joyboyisfree** [comment]
@prastut Considering that PolkaAnalytics already has many parts of the viz, why not port the entire viz as a feature under the PolkaAnalytics project? No use having 2 separate code-bases doing the same job

**Comment by prastut** [comment]
Yup. From codebase perspective chose the whatever is more efficient (users really don't care how you structure codebase). This issue is seen from a user lense.
